src/domain/collaboration/callout/creationDialog/CalloutCreationDialog.tsx

```diff
@@ -126,7 +126,11 @@ const CalloutCreationDialog: FC<CalloutCreationDialogProps> = ({
             profile: {
               displayName: callout.displayName!,
               description: callout.description!,
-              referencesData: callout.references!,
+              referencesData: callout.references?.map(ref => ({
+                name: ref.name,
+                uri: ref.uri,
+                description: ref.description,
+              }))!,
```

Comment:
I think you can use `?? []`  ( or `references!.map...` if you are sure that references is always set like on `displayName!`) instead of that exclamation mark at the end that didn't look right to me. but the result is almost the same... 
I can approve this